### PR TITLE
Feature/enhance customization support

### PR DESCRIPTION
**Pull request by TzelaDayan**

### CODE REVIEW

File: src/theme.js [hunk 1]
```diff
@@ -79,7 +79,8 @@ export const defaultTheme = {
     css: {}
   },
   MessageMedia: {
-    css: {}
+    css: {},
+    img: {}
```
Comment (ekampf) [inline src/theme.js]: undo

File: src/components/Message/MessageMedia.js [hunk 2]
```diff
@@ -10,7 +10,15 @@ const StyledMedia = styled.div`
     height:auto;
     margin:0 auto;
   }
-  ${props => props.theme.MessageMedia.css}
+
+  ${props => {
+    const img = props.theme.MessageMedia.img;
+    const css = props.theme.MessageMedia.css;
```
Comment (ekampf) [inline src/components/Message/MessageMedia.js]: undo

File: src/components/Chat.js [hunk 3]
```diff
@@ -207,14 +207,15 @@ class TextComposer extends React.Component {
   }
  
   render() {
-    const { children, svgSendIcon, displaySendButton } = this.props;
+    const { children, svgSendIcon, displaySendButton, sendText } = this.props;
```
Comment (ekampf) [inline src/components/Chat.js]: `sendText` is missing in proptypes (and defaults?)

File: src/components/Chat.js [hunk 4]
```diff
@@ -303,7 +304,8 @@ export default class Chat extends React.Component {
     onFocus: () => {},
     onBlur: () => {},
     onButtonClick: () => {},
-    displaySendButton: false
+    displaySendButton: false,
+    sendText: 'Send'
```
Comment (ekampf) [inline src/components/Chat.js]: Missing in propTypes

Comment (TzelaDayan) [reply src/components/Chat.js]: its there in line 288

**Pull request merged**